layouts/partials/header.html

```diff
@@ -2,16 +2,16 @@
     <meta charset="utf-8">
     <meta name="theme-color" content="#000" />
     <title>
-        {{ if .IsHome }}
-            {{ if .Site.Title }} 
-                {{ .Site.Title }} 
-            {{ else }}
-                {{ .Site.Params.Title }} 
-            {{ end }} 
-            &middot; {{ .Site.Params.Tagline }}
-        {{ else }}
-            {{ .Title }} &middot; {{ .Site.Title }} {{ .Site.Params.Title }}
-        {{ end }}
+        {{- if .IsHome -}}
```

Comment:
Thank you @leonrosenshein . Can you remind me what these hyphens do again?

Reply:
The go templater puts in newline characters before/after "actions" by default. Adding the `-` removes them. 

https://pkg.go.dev/text/template